changelog/unicode_properties_c.dd

```diff
@@ -0,0 +1,34 @@
+The Unicode property "C" aka "Other" has had the wrong properties associated with it.
+
+If you see this code (case insensitive), you should mitigate or fix your codebase.
+
+---
+unicode.c
+---
+
+or:
+
+---
+unicode.Other
+---
+
+This change makes it match the Unicode Techical Report #44. Unfortunately if you are already using it with its previous wrong values, this will break your code, below is a function which reflects the original values that you can use to mitigate against any breakage.
+
+---
+@property auto loadPropertyOriginal(string name)() pure
+{
+    import std.uni : unicode;
+    
```

Comment:
```suggestion

```

Reply:
There's trailing whitespace on this line

Reply:
Ah huh, I can't believe I missed that.